packit_service/worker/build/copr_build.py

```diff
@@ -233,6 +258,18 @@ def run_copr_build(self) -> TaskResults:
                 chroot=chroot,
             )
 
+        if unprocessed_chroots:
+            unprocessed = "\n".join(sorted(unprocessed_chroots))
+            available = "\n".join(sorted(self.available_chroots))
+            self.project.pr_comment(
+                pr_id=self.metadata.pr_id,
+                body="There are build targets that are not supported by COPR.\n"
+                "<details>\n<summary>Unprocessed build targets</summary>\n\n"
+                f"```\n{unprocessed}\n```\n</details>\n"
```

Comment:
this should be fairly short most of the time right? would it make sense to print it right away without the `<details>` thing?

Reply:
I'm not really sure...
- for unprocessed: I could smack there easily a lot of invalid targets, or if I wanted to be explicit just use _vim-magic_ and have a typo somewhere in there; that would also generate long list
- for supported: I just looked at the list from the code snippet, there's 42 of them